modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -113,10 +112,51 @@ Object Store v2 provides persistent storage of objects with a time to live (TTL)
 
 The tutorial provides more detailed information about creating object stores in Mule 3 and Mule 4 applications. 
 
+== Configure a Custom Time-to-Live
+
+To configure a custom time-to-live (TTL) period for keys stored in Object Store v2:
+
+. Create an app that uses Mule runtime engine version 4.2.1 or later.
+. Configure the object store in the app.
+. Select the *Store* operation in the flow and, in the properties window, click the green plus icon (+) to the right of the *Object store* field:
++
+.The arrow shows the green plus icon in the properties window.
+image::os-use-ttl-props.png[Green plus icon in the properties window]
+. Configure the TTL entries in the *Global Object Store Configuration* window:
++
+.The arrow indicates the *Entry ttl* field in the *Global Object Store Configuration* window.
+image::os-use-ttl-global-props.png[Global Object Store Configuration window]
++
+[NOTE]
+For Object Store v2, the *Max entries* field is not configurable.
++
+* *Entry ttl* 
++
+To use rolling TTL, leave this field empty.
+Otherwise, enter the custom TTL (between 0 and 2592000 seconds).
+If this value is greater than 30 days, TTL is set to the maximum value (2592000 seconds/30 days).
++
+For Mule versions earlier than 4.2.1, leave this field empty to set the TTL to the maximum (2592000 seconds).
```

Comment:
Yeah, this further complicates this definition.
If it is technically accurate, I suggest an entry similar to this:

***Entry ttl** 
+
For Mule 4.2.1 and later, enter a custom TTL value of 0 through 2592000 seconds (30 days). If you leave this field empty, you enable rolling TTL; if you exceed 2592000 seconds, the TTL defaults to 2592000 seconds. 
+
For Mule versions earlier than 4.2.1, leaving this field empty defaults to 2592000 seconds.